.changeset/clean-frogs-brake.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/create-app': minor
```

Comment:
Let's make this `patch` since we consider changes to the templates to be non-breaking changes.